presto-pinot-toolkit/src/main/java/com/facebook/presto/pinot/PinotClusterInfoFetcher.java

```diff
@@ -95,12 +107,14 @@ public PinotClusterInfoFetcher(
             JsonCodec<BrokersForTable> brokersForTableJsonCodec,
             JsonCodec<RoutingTables> routingTablesJsonCodec,
             JsonCodec<RoutingTablesV2> routingTablesV2JsonCodec,
-            JsonCodec<TimeBoundary> timeBoundaryJsonCodec)
+            JsonCodec<TimeBoundary> timeBoundaryJsonCodec,
+            JsonCodec<Instance> instanceJsonCodec)
     {
         this.brokersForTableJsonCodec = requireNonNull(brokersForTableJsonCodec, "brokers for table json codec is null");
         this.routingTablesJsonCodec = requireNonNull(routingTablesJsonCodec, "routing tables json codec is null");
         this.routingTablesV2JsonCodec = requireNonNull(routingTablesV2JsonCodec, "routing tables v2 json codec is null");
         this.timeBoundaryJsonCodec = requireNonNull(timeBoundaryJsonCodec, "time boundary json codec is null");
+        this.instanceJsonCodec = requireNonNull(instanceJsonCodec, "instance json codec is null");
```

Comment:
put this to the end to be the same order as the ones on stack and constructor parameters.